_content/gateways/gateway/faq.md

```diff
@@ -79,3 +79,9 @@ Connect to the gateways WiFi access point ``Things-Gateway-XXXX`` using the pass
 To check the status of the gateway, go to the console as directed afterwards in the activation process. Under the gateway section (https://console.thethingsnetwork.org/gateways), you can see the status. If any device sends data through your gateway, you can check it under the ‘traffic’ section.
 
 When connected to the same network, some information can be seen at http://things-gateway.local/info.
+
```

Comment:
@chiflux: This is a good addition. Minor Nitpicks.

- Please add a disclaimer that this is one of the possible solutions and that `https://github.com/TheThingsProducts/gateway/issues` is the place to look for other solutions.
- I understand that you're using `...` to describe how boring this reboot is but could you please replace it with a full stop ;).
- connector -> socket.

Thanks!

Reply:
Thanks for the review. I ammeded the wording, trying to consider your comments. Please check again.